content/webapp/components/ArchiveTree/ArchiveTree.DownloadItemRenderer.tsx

```diff
@@ -0,0 +1,71 @@
+import { FunctionComponent } from 'react';
+import { TreeControl } from '@weco/content/components/ArchiveTree/ArchiveTree.styles';
+import { UiTreeNode } from '@weco/content/components/ArchiveTree/ArchiveTree.helpers';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import DownloadItem from '@weco/content/components/WorkDetails/WorkDetails.DownloadItem';
+import { chevron, closedFolder, openFolder } from '@weco/common/icons';
+import { font } from '@weco/common/utils/classnames';
+import { controlDimensions } from './ArchiveTree.helpers';
+import { styled } from 'styled-components';
+import { TransformedCanvas } from '@weco/content/types/manifest';
+
+const ItemWrapper = styled.div.attrs({
+  className: font('intr', 6),
+})`
+  display: flex;
+  height: ${`${controlDimensions.controlHeight}px`};
```

Comment:
np but think you could just do 
```suggestion
  height: ${controlDimensions.controlHeight}px;
```